packages/utils/src/date/timestamps.ts

```diff
@@ -1,5 +1,11 @@
 export const HOURS_OF_NOON = 12;
-export const MILLISECONDS_IN_DAY = 86400000;
-export const MILLISECONDS_IN_HOUR = 3600000;
-export const MILLISECONDS_IN_MINUTE = 60000;
+export const YEARS_IN_CENTURY = 100;
+export const MONTHS_IN_YEAR = 12;
+export const HOURS_IN_DAY = 24;
+export const MINUTES_IN_HOUR = 60;
+export const SECONDS_IN_MINUTE = 60;
 export const MILLISECONDS_IN_SECOND = 1000;
+export const MILLISECONDS_IN_MINUTE = SECONDS_IN_MINUTE * MILLISECONDS_IN_SECOND;
```

Comment:
```suggestion
export const MILLISECONDS_IN_MINUTE = MILLISECONDS_IN_SECOND * SECONDS_IN_MINUTE;
```
Keeping consistent with others

Reply:
done